web3/_utils/method_formatters.py

```diff
@@ -1214,7 +1211,7 @@ def get_result_formatters(
 
 def get_error_formatters(
     method_name: Union[RPCEndpoint, Callable[..., RPCEndpoint]]
```

Comment:
Is method_name actually supposed to be a `Union[RPCEndpoint, Callable[..., RPCEndpoint]]`?

I notice `Union[RPCEndpoint, Callable[..., RPCEndpoint]]` is used throughout the codebase but the code itself seems to assume method_name is always a string value, never callable. Should this be updated as well?

Reply:
Oh, nice catch. Yeah, these can only be RPCEndpoints once they're here.